templates/index.rst

```diff
@@ -0,0 +1,27 @@
+.. LSST DM Stack  documentation master file, created by
+   sphinx-quickstart on Tue May 12 10:44:33 2015.
+   You can adapt this file completely to your liking, but it should at least
+   contain the root `toctree` directive.
+
+
+LSST DM Stack documentation
+============================
+
+Contents:
+
+.. toctree::
+
+  Task Template <template-for-tasks>
```

Comment:
Right, i did get this from your last comments, but i now cleaned up, by only linking to the Guidance Doc for the template in the toctree.